weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ChatView/ChatFormats/anthropic.ts

```diff
@@ -0,0 +1,117 @@
+import _ from 'lodash';
```

Comment:
Next step for all of these `ChatFormats` files is to use zod schema validation to automatically parse out which provider handling we should do